perma_web/perma/celery_tasks.py

```diff
@@ -1224,3 +1229,104 @@ def conditionally_queue_internet_archive_uploads_for_date_range(start_date_strin
 
     else:
         logger.info("Skipped the queuing of file upload tasks: max tasks already in progress.")
+
+
+# WACZ CONVERSION
+
+def format_time(seconds_val):
+    """
+    Converts seconds to minutes
+    """
+    if seconds_val >= 60:
+        return f"{round(seconds_val / 60, 1)} minutes"
+    else:
+        return f"{math.ceil(seconds_val)} seconds"
+
+
+def save_warc_to_temp(input_guid):
+    """
+    Gets the file path
+    Get the associated file's contents from storage
+    Saves as temp file
+    """
+    warc_storage_path = Link.objects.get(guid=input_guid).warc_storage_file()
```

Comment:
Oh! I also meant to say: since you are using the Link object here, and also at 1298, I recommend making a tweak so that we don't call out to the database twice.